front/components/assistant/Sharing.tsx

```diff
@@ -114,34 +116,50 @@ export function SharingSection({
   });
   const assistantName = agentConfiguration?.name;
 
+  console.log(agentUsage);
+
   const usageText = assistantName
     ? assistantUsageMessage({
-        assistantName,
-        usage: agentUsage.agentUsage,
-        isLoading: agentUsage.isAgentUsageLoading,
-        isError: agentUsage.isAgentUsageError,
+        assistantName: null,
+        usage: agentUsage,
+        isLoading: isAgentUsageLoading,
+        isError: isAgentUsageError,
       })
     : "";
 
   return (
-    <div className="flex flex-col gap-3">
-      <div className="text-lg font-bold text-element-900">Sharing</div>
-      <SharingDropdown
-        owner={owner}
-        agentConfiguration={agentConfiguration}
-        initialScope={initialScope}
-        newScope={newScope}
-        setNewScope={setNewScope}
-      />
-      <div className="text-sm text-element-700">
-        <div>{SCOPE_INFO[newScope].text}</div>
-        {agentUsage &&
-        agentUsage.agentUsage?.userCount &&
-        agentUsage.agentUsage.userCount > 1
-          ? usageText
-          : null}
-      </div>
-    </div>
+    <DropdownMenu>
+      <DropdownMenu.Button>
+        <Button
+          size="sm"
+          label="Sharing"
+          icon={ShapesIcon}
+          variant="tertiary"
+        />
+      </DropdownMenu.Button>
+      <DropdownMenu.Items width={319} overflow="visible">
+        <div className="flex flex-col gap-y-2 py-2">
+          <SharingDropdown
+            owner={owner}
+            agentConfiguration={agentConfiguration}
+            initialScope={initialScope}
+            newScope={newScope}
+            setNewScope={setNewScope}
+          />
+          <div className="text-sm text-element-700">
+            <div>
+              {SCOPE_INFO[newScope].text}{" "}
+              {agentUsage &&
+              agentUsage.userCount &&
+              agentUsage.userCount > 0 &&
```

Comment:
here the best IMHO would be to replace with `agentUsage.usersWithAgentInListCount` introduced ~ a week ago => fits the original framing intent in Ahuna 2, +  text does not show when creating (at a time when it does not make sense), etc.